erc20-connector/package.json

```diff
@@ -22,7 +22,7 @@
         "eslint-plugin-standard": "^4.0.0",
         "eth-gas-reporter": "^0.2.17",
         "ganache-cli": "^6.9.1",
-        "solc": "^0.5.17",
+        "solc": "0.6.2",
```

Comment:
This particular version was chosen for several reasons:

- There are some dependencies that use `0.6.2` (@openzeppelin/contracts/utils/Address.sol) so we can't use any lower version
- For larger version (>= 0.6.8) there is an issue with flat files and multiple licenses. It throw an error (read more here: https://forum.openzeppelin.com/t/solidity-0-6-8-introduces-spdx-license-identifiers/2859) (We need to file an alternative to the current tool we are using to flatten files)

Reply:
You can add the same hack that we are using [here](https://github.com/near/rainbow-bridge/blob/master/contracts/eth/dist.sh#L19).